Opc.Ua.CloudLib.Client/UACloudLibClient.cs

```diff
@@ -110,6 +129,10 @@ public UACloudLibClient(string strUsername, string strPassword) : this(StandardE
         /// <param name="strPassword">The string password.</param>
         public UACloudLibClient(string strEndpoint, string strUsername, string strPassword)
         {
```

Comment:
I've recently cleaned this class up, please merge from main.

Reply:
Merged.